src/components/biggive-main-menu/biggive-main-menu.scss

```diff
@@ -0,0 +1,417 @@
+							background-color: white;
+
+							.sub-sub-menu {
+								background-color: white;
+							}
+						}
+					}
+
+					.arrow {
+						margin-left: 5px;
+						line-height: 70px;
+						text-align: center;
+						display: inline-block;
+						color: black;
+						transition: all 0.3s ease;
+						&.transform-90 {
+							transform: rotate(90deg);
+						}
+					}
+				}
+			}
+		}
+
+		.logo {
+			min-width: 180px;
+			width: 180px;
+	
+			a {
+				font-size: 30px;
+				color: black;
+				text-decoration: none;
+				font-weight: 600;
+				display: flex;
+			}
+		}
+	
+		.bx-menu {
+			display: none;
+		}
+	}
+}
+
+@media (max-width: 1200px) {
+	nav {
+		.navbar {
+			.logo {
+				min-width: 120px;
+				width: 120px;
+			}
+			.links {
+				li {
+					padding: 0 7px;
+
+					a {
+						font-size: 15px;
+					}
+
+					.sub-menu {
+						a {
+							font-size: 13px;
+						}
+					}
+				}
+			}
+		}
+	}
+}
+
+@media (max-width:968px) {
+	.mobile-only {
+		display: block !important;
+	}
+	.row-top {
+		display: none;
+	}
+	nav {
+		.navbar {
+			.bx-menu {
+				display: block;
+				font-size: 25px;
+				color: black;
+			}
+	
+			.nav-links {
+				position: fixed;
+				top: 0;
+				left: -100%;
+				display: block;
+				max-width: 350px;
+				width: 100%;
+				background: white;
+				line-height: 40px;
+				padding: 20px;
+				box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+				transition: all 0.5s ease;
+				z-index: 1000;
+				overflow-y: scroll;
+				
+				.sidebar-logo {
+					display: flex;
+					align-items: center;
+					justify-content: space-between;
+	
+					#mobileLogo {
+						width: 130px;
+					}
+	
+					#x {
+						margin-right: 15px;
+					}
+
+					i {
+						font-size: 25px;
+						color: black;
+					}
+				}
+
+				.mobile-social-icon-wrap {
+					margin-top: 20px;
+				}
+
+				.links {
+					display: block;
+					margin-top: 20px;
+
+					li {
+						display: block;
+						border-bottom: 1px solid #e6e6e6;
+
+						.arrow {
+							line-height: 40px;
+						}
+
+						.display-sub-menu {
+							display: block !important;
+						}
+
+						.sub-menu {
+							position: relative;
+							top: 0;
+							box-shadow: none;
+							display: none;
+	
+							li {
+								border-bottom: none;
+							}
+
+							.sub-sub-menu {
+								display: none;
+								position: relative;
+								left: 0;
+		
+								li {
+									display: flex;
+									align-items: center;
+									justify-content: space-between;
+								}
+							}
+		
+							.more {
+								span {
+									display: flex;
+									align-items: center;
+								}
+
+								.sub-sub-menu {
+									&.display-sub-menu {
+										position: initial;
+									}
+								}
+
+								&:hover {
+									.sub-sub-menu {
+										display: none;
+									}
+								}
+							}
+						}
+
+						&:hover {
+							.sub-menu {
+								background-color: inherit;
+								display: none;
+
+								.sub-sub-menu {
+									background-color: inherit;
+									
```

Comment:
for this purpose can we directly detect the hover capability, instead of using screen width as a proxy for hoverability? I suppose what you say about mobile devices applies just as well to tablets. https://caniuse.com/mdn-css_at-rules_media_hover

Reply:
Good shout. Done in [2e7871e](https://github.com/thebiggive/components/pull/99/commits/2e7871e03e08de788f8e566b9e737a58ed6fa9b0)